design/protocol-changes/protocol-parameters.tex

```diff
@@ -0,0 +1,48 @@
+\section{Protocol parameters}\label{sec:protocol parameters}
+
+The Peras protocol is influenced by various new protocol parameters, also see~\cite{peras-cip}, which we list in \cref{fig:protocol parameters}.
+For explicitness and to avoid confusion, we use a relatively verbose naming scheme.
+The Cardano Ledger allows protocol parameters to be controlled via on-chain governance.
+We indicate whether this flexibility is likely to be useful.
+
+\begin{figure}[h]
+  \centering
+  \begin{tabular}{c c c c c}
+    \toprule
+    Name & Unit & Symbol & Feasible value & Governable? \\
+    \midrule
+    \perasRoundSlots{} & \unit{\slot} & $U$ & \num{90} & $\checkmark$ \\
+    \perasBlockMinSlots{} & \unit{\slot} & $L$ & $T_{\mathrm{CQ}} = \kcg/\asc$ & $\checkmark$ \\
```

Comment:
kcg is k = 2160, no? and asc = 1/20? so we're suggesting L = 43200?

Reply:
Ah thanks, I just accidentally swapped the suggested values for `perasBlockMinSlots` and `perasBlockMaxSlots`.